src/service/viewer-impl.js

```diff
@@ -130,6 +130,9 @@ export class Viewer {
     /** @private {!Observable<!JSONType>} */
     this.broadcastObservable_ = new Observable();
 
+    /** @private {!Observable<!JSONType>} */
+    this.scrollDocObservable_ = new Observable();
```

Comment:
What do you mean? Is there a PR where we are moving out the API dependencies? Do you mean changing this to the following?
```
/** @private {!Observable} */
this.scrollDocObservable_ = new Observable();
```